jinahub/segmenters/VADSpeechSegmenter/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM jinaai/jina:latest-perf
+
+RUN apt-get -y update && apt-get install -y libsndfile1
+
+# install requirements before copying the workspace
+COPY requirements.txt /requirements.txt
+RUN pip install -r requirements.txt
```

Comment:
```suggestion
RUN pip install -r requirements.txt --no-cache-dir
```

Reply:
Addressed, please see [here](https://github.com/jina-ai/executors/pull/97/commits/d6a5445f807522ab569712f843bc276be3b46a34) for the change.